client/src/components/DataFiles/DataFilesModals/DataFilesModals.scss

```diff
@@ -9,8 +9,10 @@
   .h-100 {
     padding: 0 10px;
   }
+  .breadcrumbs-container {
+    font-size: 1.2rem;
+  }
   .breadcrumbs {
-    font-size: 12px;
```

Comment:
That this now inherits is good. That's how `.as-link` buttons should work.

Gotcha:
- Not setting this to `1rem` means it becomes larger than it was in `main`.
- Design wants links in header smaller. See Dashboard table header links.

Reply:
I'm getting lost. Ignore me here.